modules/form/src/js/form.js

```diff
@@ -29,6 +29,8 @@ var FormModule = (function () {
 		var editbox = form.closest('.widget-editbox');
 		setTimeout(function() {
 			form.find('.nj-form-option').trigger('change');
+			// Reset hover title on cog wheel icon (edit widget)
+			$( editbox ).prev().prev().find(".widget-editlink").attr("title", $.fn.EasyWidgets.defaults.i18n.editTitle);
```

Comment:
Im guessing this .prev().prev() chain is needed?

Reply:
Just pushed a refactoring, $( editbox ).parent().find(...